bundle.js

```diff
@@ -48,23 +48,29 @@
         displayHeadlines() {
           const headlines = this.model.getHeadlines();
           headlines.forEach((headline) => {
-            this.headlinesContainerEl = document.querySelector("#headline-container");
-            const headlineEl = document.createElement("div");
-            headlineEl.innerText = headline["headline"];
-            headlineEl.className = "headline";
-            headlineEl.style.cssText = "font-size:25px;color:darkblue";
-            this.headlinesContainerEl.append(headlineEl);
-            const viewArticleButtonEl = document.createElement("button");
-            viewArticleButtonEl.innerText = "View Article";
-            this.headlinesContainerEl.append(viewArticleButtonEl);
-            const imageEl = document.createElement("img");
-            imageEl.className = "headlineImage";
-            imageEl.src = headline["thumbnail"];
-            this.headlinesContainerEl.append(imageEl);
+            this.displayHeadline(headline);
+            this.displayViewArticleButton();
+            this.displayImage(headline);
           });
         }
-        setImageSrc(data) {
-          data;
+        displayHeadline(headline) {
+          this.headlinesContainerEl = document.querySelector("#headline-container");
+          const headlineEl = document.createElement("div");
+          headlineEl.innerText = headline["headline"];
+          headlineEl.className = "headline";
+          headlineEl.style.cssText = "font-size:25px;color:darkblue";
+          this.headlinesContainerEl.append(headlineEl);
+        }
+        displayViewArticleButton() {
```

Comment:
Same as above, well done for separating the logic to display different elements!